src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/JSException.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.Versioning;
+
+namespace System.Runtime.InteropServices.JavaScript
+{
+    /// <summary>
+    /// Represents an Exception initiated from the JavaScript interop code.
+    /// </summary>
+    [SupportedOSPlatform("browser")]
+    public sealed class JSException : Exception
+    {
+        // TODO after https://github.com/dotnet/runtime/issues/70133
+
+        public JSException(string msg) : base(msg)
+        {
+        }
+
+        public override string ToString()
+        {
+            // skip the expensive stack trace
+            return Message;
```

Comment:
Should we really be doing this? What code path requires us to do it?

Reply:
Debugger maybe ?